README.markdown

```diff
@@ -22,29 +22,50 @@ The ETE is released under the [MIT License](LICENSE.txt).
 ## Installation
 
 Installing ETEngine on a local machine can be a bit involved, owing to the
-number of dependencies. Fortunately, most can be installed automatically using
-Ruby's "Bundler" library, which is required to use ETEngine.
+number of dependencies. Assuming you can run a 'normal' rails application on your local machine,
+you have to follow these steps to run ET-Engine.
+
+1. Install the "Graphviz" library
+   * Mac users with [Homebrew][homebrew]: `brew install graphviz`
+   * Ubuntu: `sudo apt-get install graphviz libgraphviz-dev`
+
+1. Install "MySQL" server
+   * Mac: Install latest version using the [Native Package][mysql] (choose the 64-bit DMG version)
+   * Ubuntu: `sudo apt-get install mysql-server-5.5 libmysqlclient-dev`
 
-1. Install the "Graphviz" library; for Mac users with [Homebrew][homebrew], this
-   is as simple as `brew install graphviz`.
+1. Clone this repository with `git clone git@github.com:quintel/etengine.git`
 
-2. Run `bundle install` to install the dependencies required by ETEngine.
+1. Run `bundle install` to install the dependencies required by ETEngine.
 
-3. Copy "config/config.sample.yml" to "config/config.yml", and
-   "config/database.sample.yml" to "config/database.yml", making any changes --
-   particularly to the database configuration -- as you see fit.
+1. Create your personal configuration files from the samples with
+   ```
+   cp -vn config/database.sample.yml config/database.yml
+   cp -vn config/config.sample.yml config/config.yml
+   ```
+   Then make any changes -- particularly to the database configuration -- as you see fit.
+   * Probably set "standalone" to `true` in "config/config.yml"
 
-4. Clone a copy of [ETSource][etsource] –– which contains the data for each
-   region. Edit "config/config.yml" and enter the ETSource directory into the
-   "etsource_export" and "etsource_working_copy" options.
+1. Clone a copy of [ETSource][etsource] –– which contains the data for each
+   region:
+   1. `cd ..; git clone git@github.com:quintel/etsource.git`
```

Comment:
A local copy of ETSource is needed to run ETEngine, so I think that line makes sense.

There isn't any setup required to use ETSource with ETEngine other than cloning the repo. If you want to make *changes* to ETSource then that's another matter; you'd want to run `bundle install` so that the tests can be performed, however I don't think that's a necessary part of setting up ETEngine.